src/content/reference/react/useSyncExternalStore.md

```diff
@@ -57,6 +57,10 @@ The current snapshot of the store which you can use in your rendering logic.
 
 * If a different `subscribe` function is passed during a re-render, React will re-subscribe to the store using the newly passed `subscribe` function. You can prevent this by declaring `subscribe` outside the component.
 
+* If the store is mutated during a [non-blocking transition update](/reference/react/useTransition), React will fall back to performing that update as blocking. Specifically, React will call `getSnapshot` a second time just before applying changes to the DOM. If it returns a different value than when it was called originally, React will restart the transition update from scratch, this time applying it as a blocking update, to ensure that every component on screen is reflecting the same version of the store.
```

Comment:
Is this only for transitions, or also useDeferredValue, and eventually useFormState and unOptimistic? Would it be better to talk about "non urgent updates" instead of transitions specifically?

Reply:
I’m not picky about the nomenclature here. I usually call them transition updates because I think we call it transition priority internally IIRC but fine with whatever. I wish we had a page that talked about update priorities generally.